tools/clang/include/clang/AST/Decl.h

```diff
@@ -810,9 +810,14 @@ class VarDecl : public DeclaratorDecl, public Redeclarable<VarDecl> {
     /// declared.
     unsigned ScopeDepthOrObjCQuals : 7;
 
+    // HLSL Change start
     /// Whether the parameter is copied out.
     unsigned IsModifierOut : 1;
```

Comment:
This was an existing HLSL addition to this bitfield that made me think I could add the index here too.